modules/sdk/gradle-plugins-workspace/build.gradle

```diff
@@ -231,6 +231,10 @@ gradleTest {
 	gradleArguments "--project-prop", "pluginClasspathDir=${uri(gradleTestDependenciesDir)}"
 
 	versions gradleVersion
+
+	beforeTest {
```

Comment:
Need to remove?

Reply:
I left it there intentionally, but I realize we don't really have statements like this elsewhere. To me it seemed like reasonable basic output - it's comparable to what we get when we run our unit/integration tests. I won't fight too hard on it though :)